Cargo.toml

```diff
@@ -61,7 +61,7 @@ serde = { version = "1.0", optional = true }
 packed_simd = { version = "0.1.0", features = ["into_bits"], optional = true }
 
 [features]
-nightly = ["subtle/nightly", "clear_on_drop/nightly"]
+nightly = ["subtle/nightly", "clear_on_drop/nightly", "byteorder/i128"]
```

Comment:
Why do we want to feature-gate our `u128` support on `nightly`? They're stable now. Why not bump the byteorder version instead?